batch/src/main/java/com/hartwig/batch/operations/rna/RnaStarMapping.java

```diff
@@ -0,0 +1,177 @@
+package com.hartwig.batch.operations.rna;
+
+import static java.lang.String.format;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.google.common.collect.Lists;
+import com.hartwig.batch.BatchOperation;
+import com.hartwig.batch.input.InputBundle;
+import com.hartwig.batch.input.InputFileDescriptor;
+import com.hartwig.batch.operations.OperationDescriptor;
+import com.hartwig.pipeline.ResultsDirectory;
+import com.hartwig.pipeline.calling.command.VersionedToolCommand;
+import com.hartwig.pipeline.execution.vm.Bash;
+import com.hartwig.pipeline.execution.vm.BashCommand;
+import com.hartwig.pipeline.execution.vm.BashStartupScript;
+import com.hartwig.pipeline.execution.vm.OutputUpload;
+import com.hartwig.pipeline.execution.vm.RuntimeFiles;
+import com.hartwig.pipeline.execution.vm.SambambaCommand;
+import com.hartwig.pipeline.execution.vm.VirtualMachineJobDefinition;
+import com.hartwig.pipeline.execution.vm.ImmutableVirtualMachineJobDefinition;
+import com.hartwig.pipeline.execution.vm.VirtualMachinePerformanceProfile;
+import com.hartwig.pipeline.execution.vm.VmDirectories;
+import com.hartwig.pipeline.storage.GoogleStorageLocation;
+import com.hartwig.pipeline.storage.RuntimeBucket;
+import com.hartwig.pipeline.tools.Versions;
+
+public class RnaStarMapping implements BatchOperation {
+
+    private static final String REF_GENCODE_37 = "/opt/resources/hs37d5_GENCODE19";
+    // private static final String RNA_FASTQ_FILE_LIST = "rna_sample_fastqs_20200318.txt";
+
+    @Override
+    public VirtualMachineJobDefinition execute(
+            InputBundle inputs, RuntimeBucket bucket, BashStartupScript startupScript, RuntimeFiles executionFlags) {
+
+        InputFileDescriptor descriptor = inputs.get();
+
+        final String batchInputs = descriptor.value();
+        final String[] batchItems = batchInputs.split(",");
+
+        if(batchItems.length != 2)
+        {
+            System.out.print(String.format("invalid input arguments(%s) - expected SampleId,PathToFastqFiles", batchInputs));
+            return null;
+        }
```

Comment:
I think you want to make use of the ability to use JSON inputs here, rather than encoding a bunch of inputs into a CSV list. To do that you'll change the `InputType` in the `OperationDescriptor`. Then you'll be able to specify multiple inputs for each job.